src/components/Messages/Messages.js

```diff
@@ -1,13 +1,18 @@
 import React from "react";
+import { fetchChannelMessages } from "../../utils/fetcher";
+
+export const Messages = ({ currentChannel }) => {
+  let messages = fetchChannelMessages(currentChannel);
 
-const Messages = () => {
   return (
     <div>
-      <header>
-        <h1>Messages</h1>
-      </header>
+      <h1>{currentChannel}</h1>
+      <p>Lorem ipsum</p>
+      <div>
+        {messages.map(message => {
+          return <li>{message}</li>;
```

Comment:
I would just use a `div` also you're using `li` outside of a `ol`|`ul`, I'm shocked this doesn't break. In addition, you don't need to removed the `li` style, like the bullet point or number it automatically adds.